picard/ui/coverartbox.py

```diff
@@ -169,16 +177,27 @@ def fetch_remote_image(self, url):
                     data = f.read()
                 self.load_remote_image(url, mime, data)
 
-    def on_remote_image_fetched(self, url, data, reply, error):
+    def on_remote_image_fetched(self, url, data, reply, error, fallback_data=None):
         mime = reply.header(QtNetwork.QNetworkRequest.ContentTypeHeader)
         if mime in ('image/jpeg', 'image/png'):
             self.load_remote_image(url, mime, data)
-        elif reply.url().hasQueryItem("imgurl"):
+        elif url.hasQueryItem("imgurl"):
```

Comment:
Probably, yes, but here it's used to test for Google Images' use of the imgurl query item, and that's lost in the redirect, so url() (and not reply.url() ) has to be used.